mockfirestore/document.py

```diff
@@ -6,6 +6,7 @@
 from mockfirestore._helpers import (
     Timestamp, Document, Store, get_by_path, set_by_path, delete_by_path, get_document_iterator
 )
+from google.api_core.exceptions import NotFound
```

Comment:
The `update()` method uses the `NotFound` that's defined in the library (the import on line 5), so this import isn't needed.

Reply:
Thanks! Fixed in 8701374